operator/model/src/main/java/io/apicurio/registry/operator/api/v3/status/Conditions.java

```diff
@@ -0,0 +1,160 @@
+package io.apicurio.registry.operator.api.v3.status;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.annotation.JsonSetter;
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.fasterxml.jackson.annotation.Nulls;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import io.fabric8.generator.annotation.Min;
+import io.fabric8.generator.annotation.Pattern;
+import io.fabric8.generator.annotation.Required;
+import io.fabric8.kubernetes.api.model.KubernetesResource;
+
+import java.time.ZonedDateTime;
+
+@JsonInclude(com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({ "lastTransitionTime", "message", "observedGeneration", "reason", "status", "type" })
+@JsonDeserialize(using = JsonDeserializer.None.class)
+public class Conditions implements KubernetesResource {
+
+    /**
+     * lastTransitionTime is the last time the condition transitioned from one status to another. This should
+     * be when the underlying condition changed. If that is not known, then using the time when the API field
+     * changed is acceptable.
+     */
+    @JsonProperty("lastTransitionTime")
+    @Required()
+    @JsonPropertyDescription("lastTransitionTime is the last time the condition transitioned from one status to another. This should be when the underlying condition changed.  If that is not known, then using the time when the API field changed is acceptable.")
+    @JsonSetter(nulls = com.fasterxml.jackson.annotation.Nulls.SKIP)
+    private ZonedDateTime lastTransitionTime;
+
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ssVV")
+    public ZonedDateTime getLastTransitionTime() {
+        return lastTransitionTime;
+    }
+
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss[XXX][VV]")
```

Comment:
Why there are two different formats for getter and setter?

Reply:
A long long story here, basically:
https://github.com/fabric8io/kubernetes-client/pull/5384#issuecomment-1675164888

happy to share the full story when needed XD